doc/schema-migration.md

```diff
@@ -0,0 +1,52 @@
+# Schema Migration or Norms
+
+Schema migration with Datahike is the evolution of your current schema into a future schema.
+We are calling the schema migrations 'norms' to avoid misunderstandings with a migration
+from an older version of Datahike to a newer version.
+
+When we are speaking of changes to your schema, these should always add new definitions and
```

Comment:
So can I just transact new schema entries at any point or do I need to worry about norms now? This is not clear from this description. I hope it is the first, since monotonic schema additions should stay painless, I think.